test/pkgs/NestedPackage/pkg.json

```diff
@@ -0,0 +1 @@
+{"license":"","file_hash":null,"name":"NestedPackage","version":"1.0.0","description":"original package","group":"","keywords":null,"dependencies":[],"contents":"","engine_version":"2.1.0.7840","engine":"dynamo","engine_metadata":"","site_url":"","repository_url":"","contains_binaries":true,"node_libraries":["Package, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"]}
```

Comment:
any reason you need any entry point at all in node libraries?

Reply:
No reason, just copied a different package from test folder for this